packages/button/src/Button/Button.tsx

```diff
@@ -22,13 +22,15 @@ export const Button = React.forwardRef(function Button(
     darkMode: darkModeProp,
     baseFontSize = BaseFontSize.Body1,
     disabled = false,
+    isLoading = false,
```

Comment:
Searching in regex for `is[A-Z]` across all `*.types.ts` files shows we have more boolean props with the `is_` prefix than not. I think it's also good practice to indicate clearly that it's a boolean state prop and not anything else. 

I would argue that `open` not being `isOpen` is the anti-pattern that should be addressed. It feels like common sense to me that event handlers should be prefixed with `on_`, booleans with `is_`, and component props with capital letters unless the prop name makes it impossible to confuse. With `open`, `open={true}` could mean `isOpen`, `openable`, etc.

Reply:
+1 to using `is_` for booleans. Much clearer that it's referring to a boolean. Cases like `open` can be confusing, since it's unclear whether it's a state variable or a function to set the state

Reply:
Can someone add this to our API style guide please

Reply:
It's already [there](https://github.com/mongodb/leafygreen-ui/blob/main/STYLEGUIDE.md#use-be-verbs-for-boolean-types-eg-shouldiscandoeshas)